vuepress/docs/next/docs/reference/deployment-structure.md

```diff
@@ -51,37 +50,35 @@ A customized image can include:
 * New APIs 
 * Legacy Entando plugins
 * New database tables  
-* Other extensions to the **entando-core** 
+* Other extensions to the entando-core
 
-It is recommended that most extensions to the Entando Platform occur in microservices. However, legacy 
+It is recommended that microservices are used for most extensions to the Entando Platform. However, legacy 
 integrations, extensions to the CMS, and migrations from earlier Entando versions may require changes to the **entando-de-app**. 
 
 ## App Builder
-The App Builder is the user-friendly frontend UI for the **entando-de-app**. A ReactJS application, it is served via node in the default deployment. In a quickstart deployment, the App Builder container is deployed in the **entando-app** multi-container pod. It communicates with the **entando-de-app** and the Entando Component Manager (ECM) via [REST 
-APIs](../consume/entando-apis.md).  The ECM provides information about bundles deployed to the Local Hub.
+The **App Builder** is the user-friendly frontend UI for the `entando-de-app`. A ReactJS application, it is served via Node in the default deployment. It communicates with the `entando-de-app` and the Entando Component Manager via [REST 
+APIs](../consume/entando-apis.md). The Component Manager provides information about bundles deployed to the Local Hub.
 
 * GitHub: <https://github.com/entando/app-builder/>
 * DockerHub: <https://hub.docker.com/repository/docker/entando/app-builder/>
 
 #### Customization
 The App Builder is customized as part of many Entando implementations. 
-It can be customized at runtime via micro frontends 
-[widget configuration](../../tutorials/create/mfe/widget-configuration.md).
+It can be customized via [configuration micro frontends](../../tutorials/create/mfe/widget-configuration.md) and with [Entando Platform Capabilities (EPCs)](../../tutorials/create/mfe/epc.md).
 
-## Entando Component Manager (ECM)
-The ECM provides the link between the **entando-de-app**, or your custom core instance, and the ECR. The ECM queries the Entando Kubernetes service to fetch available 
+## Entando Component Manager 
+The **Entando Component Manager** provides the link between the `entando-de-app`, or your custom core instance, and the Local Hub. It queries the Entando Kubernetes service to fetch available 
 bundles that have been deployed as custom resources inside the cluster. 
 
-The ECM also manages relationships between an Entando Application and the 
-installed plugins. This can be seen in the plugin link custom resource in Kubernetes. 
+The Component Manager also administers the relationships between an Entando Application and the 
+installed plugins. This is seen in the plugin link custom resource in Kubernetes. 
```

Comment:
"plugin link custom resource" is so many modifiers... "custom resource for the plugin link"